### ISSUE TITLE
rnn language model example https://www.tensorflow.org/versions/r0.8/tutorials/recurrent/index.html isnt working

### ISSUE THREAD

**Issue by dexter4455** [tensorflow/tensorflow#3036]
GitHub issues are for bugs / installation problems / feature requests.  
For general support from the community, see [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow).
To make bugs and feature requests more easy to find and organize, we close issues that are deemed
out of scope for GitHub Issues and point people to StackOverflow.

For bugs or installation issues, please provide the following information.
The more information you provide, the more easily we will be able to offer
help and advice.

### Environment info
Operating System:
Ubuntu 14.04
Installed version of CUDA and cuDNN: cuda 7.5, cudnn 5.0.5
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):
-rw-r--r-- 1 root root 189170 Jun 12 02:51 /usr/local/cuda/lib/libcudadevrt.a
lrwxrwxrwx 1 root root     16 Jun 12 02:51 /usr/local/cuda/lib/libcudart.so -> libcudart.so.7.5
lrwxrwxrwx 1 root root     19 Jun 12 02:51 /usr/local/cuda/lib/libcudart.so.7.5 -> libcudart.so.7.5.18
-rwxr-xr-x 1 root root 311596 Jun 12 02:51 /usr/local/cuda/lib/libcudart.so.7.5.18
-rw-r--r-- 1 root root 558020 Jun 12 02:51 /usr/local/cuda/lib/libcudart_static.a

If installed from binary pip package, provide:

1. Which pip package you installed.
2. The output from `python -c "import tensorflow; print(tensorflow.__version__)"`.
I tensorflow/stream_executor/dso_loader.cc:105] successfully opened CUDA library libcublas.so.7.5 locally
I tensorflow/stream_executor/dso_loader.cc:105] successfully opened CUDA library libcudnn.so.5.0.5 locally
I tensorflow/stream_executor/dso_loader.cc:105] successfully opened CUDA library libcufft.so.7.5 locally
I tensorflow/stream_executor/dso_loader.cc:105] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:105] successfully opened CUDA library libcurand.so.7.5 locally
0.8.0
If installed from sources, provide the commit hash:

### Steps to reproduce
1. example from https://www.tensorflow.org/versions/r0.8/tutorials/recurrent/index.html github in: tensorflow/tensorflow/models/rnn/ptb/ 
2. wget http://www.fit.vutbr.cz/~imikolov/rnnlm/simple-examples.tgz and extract it
3. python ptb_word_lm.py --data_path=simple-examples/data/ after you get 

### What have you tried?
1. Running it - i'm new to TF, however the cost is nan and doesn't make sense that iters is nan as well. Tried both TF 0.9 and 0.8 , same output 

### Logs or other output that would be helpful
(If logs are large, please upload as attachment).
0.004 perplexity: 1.000 speed: 9537 wps
0.104 perplexity: nan speed: 11236 wps
0.204 perplexity: nan speed: 11232 wps
0.304 perplexity: nan speed: 11206 wps
0.404 perplexity: nan speed: 11231 wps

0.504 perplexity: nan speed: 11220 wps
0.604 perplexity: nan speed: 11239 wps
0.703 perplexity: nan speed: 11250 wps
0.803 perplexity: nan speed: 11259 wps
0.903 perplexity: nan speed: 11261 wps
Epoch: 4 Train Perplexity: nan
Epoch: 4 Valid Perplexity: nan
Epoch: 5 Learning rate: 1.000
0.004 perplexity: 1.000 speed: 9563 wps
0.104 perplexity: nan speed: 11171 wps
0.204 perplexity: nan speed: 11219 wps
0.304 perplexity: nan speed: 11245 wps
0.404 perplexity: nan speed: 11264 wps
0.504 perplexity: nan speed: 11269 wps
0.604 perplexity: nan speed: 11265 wps
0.703 perplexity: nan speed: 11269 wps

**Comment by prb12** [comment]
Possibly related to #2037 ?

@dexter4455 Could you please let me know your GPU hardware / driver config.
Also, it might be useful if you could provide the output of:
ls -lR simple-examples/data

**Comment by dexter4455** [comment]
@prb12 thanks for replying!
GPU: Amazon EC2 K520 GRID or laptop: Mac book pro

output:
```
-rw-r--r-- 1 ubuntu ubuntu   884846 Feb 23  2011 ptb.char.test.txt
-rw-r--r-- 1 ubuntu ubuntu 10034964 Feb 23  2011 ptb.char.train.txt
-rw-r--r-- 1 ubuntu ubuntu   786084 Feb 23  2011 ptb.char.valid.txt
-rw-r--r-- 1 ubuntu ubuntu   449945 Aug  6  2010 ptb.test.txt
-rw-r--r-- 1 ubuntu ubuntu  5101618 Aug  6  2010 ptb.train.txt
-rw-r--r-- 1 ubuntu ubuntu   399782 Aug  6  2010 ptb.valid.txt
-rw-r--r-- 1 ubuntu ubuntu      609 Sep 13  2011 README
```

**Comment by dexter4455** [comment]
@prb12 any update?
or anyone else? what to do next...

**Comment by dexter4455** [comment]
switched to 0.9 and things worked

**Comment by prb12** [comment]
@zheng-xq  Very difficult to repro - but possibly related to some issues you mentioned.  Given it appears fixed by 0.0, please close if there is not helpful information here.

**Comment by zheng-xq** [comment]
This should be a different issue from another failure that we were investigating. Closing for now.
